config/supervisord.conf

```diff
@@ -76,10 +76,10 @@ startretries=100
 startsecs=1
 autostart=true
 
-[program:scorebot]
-command=python -m rcon.scorebot
-environment=LOGGING_FILENAME=scorebot_%(ENV_SERVER_NUMBER)s.log
-startretries=24
+[program:scoreboard]
```

Comment:
When doing a release, this should be explicitly mentioned. Some people (including me) have their "own" supervisord.conf, where they need to make these changes well